sdk/tables/Azure.Data.Tables/src/TableRestClient.cs

```diff
@@ -38,6 +38,7 @@ internal HttpMessage CreateBatchRequest(MultipartContent content, string request
             {
                 request.Headers.Add("Prefer", responsePreference.Value.ToString());
             }
+            request.Headers.Add("Accept", "application/json");
```

Comment:
I could have sworn there was a content type class with these constants somewhere in `Azure.Core`, but I can't find it.

Reply:
They are all string literals in generated code also.